extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractor.java

```diff
@@ -0,0 +1,205 @@
+package org.schabi.newpipe.extractor.services.youtube.extractors;
+
+import java.io.IOException;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.schabi.newpipe.extractor.StreamingService;
+import org.schabi.newpipe.extractor.downloader.Downloader;
+import org.schabi.newpipe.extractor.downloader.Response;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.linkhandler.LinkHandlerFactory;
+import org.schabi.newpipe.extractor.linkhandler.ListLinkHandler;
+import org.schabi.newpipe.extractor.localization.TimeAgoParser;
+import org.schabi.newpipe.extractor.playlist.PlaylistExtractor;
+import org.schabi.newpipe.extractor.services.youtube.linkHandler.YoutubeParsingHelper;
+import org.schabi.newpipe.extractor.stream.StreamInfoItem;
+import org.schabi.newpipe.extractor.stream.StreamInfoItemsCollector;
+
+/**
+ * A YoutubePlaylistExtractor for a mix (auto-generated playlist).
+ * It handles urls in the format of "youtube.com/watch?v=videoId&list=playlistId"
+ */
+public class YoutubeMixPlaylistExtractor extends PlaylistExtractor {
+
+  private Document doc;
+
+  public YoutubeMixPlaylistExtractor(StreamingService service, ListLinkHandler linkHandler) {
+    super(service, linkHandler);
+  }
+
+  @Override
+  public void onFetchPage(@Nonnull Downloader downloader) throws IOException, ExtractionException {
+    final String url = getUrl();
+    final Response response = downloader.get(url, getExtractorLocalization());
+    doc = YoutubeParsingHelper.parseAndCheckPage(url, response);
+  }
+
+  @Nonnull
+  @Override
+  public String getName() throws ParsingException {
+    try {
+      return doc.select("div[class=\"playlist-info\"] h3[class=\"playlist-title\"]").first().text();
+    } catch (Exception e) {
+      throw new ParsingException("Could not get playlist name", e);
+    }
+  }
+
+  @Override
+  public String getThumbnailUrl() throws ParsingException {
+    try {
+      Element li = doc.select("ol[class*=\"playlist-videos-list\"] li").first();
+      String videoId = li.attr("data-video-id");
+      if (videoId != null && !videoId.isEmpty()) {
+        //higher quality
+        return getThumbnailUrlFromId(videoId);
+      } else {
+        //lower quality
+        return doc.select("ol[class*=\"playlist-videos-list\"] li").first()
+            .attr("data-thumbnail-url");
+      }
+    } catch (Exception e) {
+      throw new ParsingException("Could not get playlist thumbnail", e);
+    }
+  }
+
+  @Override
+  public String getBannerUrl() {
+    return "";
+  }
+
+  @Override
+  public String getUploaderUrl() {
+    //Youtube mix are auto-generated
+    return "";
+  }
+
+  @Override
+  public String getUploaderName() {
+    //Youtube mix are auto-generated
+    return "";
```

Comment:
return YouTube instead of nothing.
See this screenshot, on YouTube website (look at top left):
<img alt="YouTube_mix" src="https://user-images.githubusercontent.com/58657617/73613027-63115c00-45e9-11ea-82e3-b64ba0c04dd0.png" width="1000">

Reply:
I don't have that YouTube string marked in you image when i open the site.

But that could also be changed in the app, since there was already a PR to deal with playlists without uploader. Just the String resource would need to be changed.
TeamNewPipe/NewPipe#2724